mesonbuild/dependencies/misc.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, environment, kwargs):
                     self.version = pkgdep.get_version()
                     self.is_found = True
                     self.pcdep.append(pkgdep)
+            if self.is_found:
+                return
```

Comment:
Shouldn't we return in the if block within the for loop instead?

Reply:
It needs to pickup both of them, this is one of the many packages with broken pkg-config .pc files, where the dependency between Fortran and C is not in the .pc files as it "should" be.